packages/nextjs/components/blockexplorer/TransactionsTable.tsx

```diff
@@ -0,0 +1,139 @@
+import Link from "next/link";
+import { TransactionResponse } from "@ethersproject/abstract-provider";
+import { ethers } from "ethers";
+import { Address } from "~~/components/scaffold-eth";
+
+type TransactionWithFunction = TransactionResponse & { functionName?: string };
+
+// Define your types
+interface Block {
+  hash: string;
+  transactions: TransactionWithFunction[];
+  timestamp: number;
+  number: number;
+}
+
+interface ConfiguredNetwork {
+  nativeCurrency: {
+    symbol: string;
+  };
+}
+
+interface TransactionReceipts {
+  [key: string]: {
+    contractAddress: string;
+  };
+}
+
+interface TransactionsTableProps {
+  blocks: Block[];
+  transactionReceipts: TransactionReceipts;
+  configuredNetwork: ConfiguredNetwork;
+}
+
+export const TransactionsTable: React.FC<TransactionsTableProps> = ({
```

Comment:
We are avoiding the use of `React.FC` checkout https://github.com/scaffold-eth/scaffold-eth-2/pull/36#discussion_r1015652259 and https://github.com/scaffold-eth/scaffold-eth-2/pull/35#discussion_r1008105245

Instead you can do : 

```ts
{ blocks, transactionReceipts } : {blocks: Block[];  transactionReceipts: TransactionReceipts; } 
```

Or checkout this -> https://github.com/scaffold-eth/scaffold-eth-2/blob/dd5082c400f07e7ce98879770290e4d6a49ddfa0/packages/nextjs/components/scaffold-eth/Address.tsx#L19